rules.d/99-systemd.rules.in

```diff
@@ -20,6 +20,8 @@ SUBSYSTEM=="mtd", TAG+="systemd"
 
 SUBSYSTEM=="ubi", TAG+="systemd"
 
+SUBSYSTEM=="input", TAG+="systemd"
```

Comment:
what's the rationale here?

Reply:
As noted in the PR summary, for my use case I want to be able to do `After=` and `BindsTo=` the device unit corresponding to my Yubikey. That way I can have a stoppable oneshot service that does some work when the Yubikey is attached and detached.

My understanding is that this is the correct way to make sure the .device unit is generated.

Reply:
hmm, well, but not sure we should do that for all input devices.

i think it might be ok to add that for devices tagged as security devices, but not for all input devices.